src/index.html

```diff
@@ -9,6 +9,41 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo_link">
```

Comment:
Following the BEM naming convention please add a double underscore instead of one for block elements styles `class="logo__link"`

Please change that in the whole project HTML and CSS :)